rs/pocket_ic_server/src/lib.rs

```diff
@@ -91,3 +93,28 @@ pub fn copy_dir(
     }
     Ok(())
 }
+
+#[derive(Clone, Debug)]
+pub struct SubnetBlockmakerMetrics {
```

Comment:
```suggestion
pub struct SubnetBlockmaker {
```
this struct does not carry any metrics (it is the mgmt canister that assembles metrics from a _sequence_ of such payloads and serves them via the node_metrics_history endpoint)